android/assets/jsons/Units.json

```diff
@@ -25,6 +25,7 @@
 		movement:2,
 		strength:5,
 		cost:25,
+		obsoleteTech:"Scientific Theory",
```

Comment:
scout should have obsoleteTech.

Reply:
Nothing really replaces it as far as "no movement penalties" goes.
Also, not obsoleted in original civ:
https://civilization.fandom.com/wiki/Scout_(Civ5)

Reply:
you can find the words "The Scout becomes obsolete (unprocurable) with the discovery of Scientific Theory." in your given webpage.

Reply:
Now I see it, you are right and I was wrong :)